nemo/collections/asr/data/audio_to_text_dataset.py

```diff
@@ -22,6 +23,7 @@
 from torch.utils.data import ChainDataset
 
 from nemo.collections.asr.data import audio_to_text, audio_to_text_dali
+from nemo.collections.asr.data.audio_to_text import RandomizedChainDataset
 from nemo.utils import logging
```

Comment:
Don't import like this, use the above two imports.

Reply:
fixed.